cmake/OpenGeodeIO.cmake

```diff
@@ -26,7 +26,7 @@ option(OPENGEODEIO_WITH_TESTS "Compile test projects" ON)
 #------------------------------------------------------------------------------------------------
 # Platform dependent settings
 if(UNIX)
-    add_compile_options(-Wall -Wextra -Wno-attributes)
+    add_compile_options(-Wall -Wextra -Wno-attributes -fvisibility=hidden)

```

Comment:
set(CMAKE_CXX_VISIBILITY_PRESET hidden), maybe we should try to add this line in the OpenGeode/cmake/utils.cmake